src/tracks/playabletrack/wavetrack/ui/SpectrumCache.cpp

```diff
@@ -374,6 +373,8 @@ void SpecCache::Populate
       const int lowerBoundX = jj == 0 ? 0 : copyEnd;
       const int upperBoundX = jj == 0 ? copyBegin : numPixels;
 
+// todo(mhodgkinson): I don't find an option to define _OPENMP anywhere. Is this
+// still of interest?
```

Comment:
I recall this was some non-team member's experiment that wasn't completed
See commit 8e1b084f01f97266eb843af72ec8d31045aa11fa

Reply:
Mmh, but this commit is just making use of it in some other file. In any case, if you're correct now one should care if I removed those ifs. Will append a tentative commit that does that.